.editorconfig

```diff
@@ -3,24 +3,15 @@ root = true
 [*]
 charset = utf-8
 end_of_line = lf
+indent_size = 4
 indent_style = tab
 insert_final_newline = true
-
-[*.{cpp,hpp,c,h,mm}]
+max_line_length = 120
 trim_trailing_whitespace = true
 
-[{*.gradle,AndroidManifest.xml}]
-indent_style = space
-indent_size = 4
-
 [{*.py,SConstruct,SCsub}]
 indent_style = space
-indent_size = 4
 
-# YAML requires indentation with spaces instead of tabs.
 [{*.{yml,yaml},.clang{-format,-tidy,d}}]
-indent_style = space
 indent_size = 2
-
-[*.svg]
-insert_final_newline = false
```

Comment:
This will change behavior, our optimized SVG files don't have final newline (at least some of them I checked, like `Node.svg`).

That's a bit of an oddity in the repo but it seems to be done on purpose by the svg optimizer we use to save even just one extra byte.

Reply:
Riiiight, I remember this. Before svgo it was inconsistently applied throughout the repo, and the optimizer removes the newlines by default. I thiiiink the svg tool can be setup to put in a final newline, but it's not a pertinent change. Maybe something to revisit if the tool every gets updated, though I'll just keep this line for now